appendix/migrate-to-postgresql.rst

```diff
@@ -1,10 +1,9 @@
 Migrate to PostgreSQL Server
 ============================
 
-.. include:: /appendix/includes/mysql-deprication-note.rst
-
-The following guide will provide you with a rough direction through that
-migration process.
+The following guide will give you some hints for the migration from
```

Comment:
What do you think, should we mention that for existing mysql systems they first need to be migrated *before* updating to Zammad 7.0?